backend/app/services/implementations/log_records_service.py

```diff
@@ -158,6 +173,14 @@ def filter_log_records(self, filters=None):
                             sql = sql + "\nAND " + options[filter](filters.get(filter))
         return sql
     
+    def join_resident_attributes(self):
+        return "LEFT JOIN\n \
+                    (SELECT logs.log_id, string_to_array(string_agg(CAST(residents.id AS VARCHAR(10)), ','), ',') AS resident_ids, string_to_array(string_agg(CONCAT(residents.initial, residents.room_num), ','), ',') AS residents FROM log_records logs\n \
```

Comment:
Instead of using `string_to_array(string_agg())`, you should be able to use ARRAY_AGG() instead to match the same functionality

Also I'm curious, is there a reason why you casted `residents.id` to a `VARCHAR`?